components/document/src/_mixin.scss

```diff
@@ -3,8 +3,15 @@
  * Copyright (c) 2021 Robbert Broersma
  */
 
-@mixin utrecht-document {
+@mixin reset-font-smoothing() {
```

Comment:
I don't think we need the `()` after `reset-font-smoothing` we do not use it anywhere else :)